lib/jobs/secure-erase-drive.js

```diff
@@ -407,7 +407,7 @@ function secureEraseJobFactory(
         }, []);
 
         // add script's url in the first command
-        this.commands[0].downloadUrl = '/api/1.1/templates/secure_erase.py';
+        this.commands[0].downloadUrl = '/api/current/templates/secure_erase.py';
```

Comment:
I think it would be easier to use some of the built-in API render variables we have, so that we only have to change this in one place in the future, e.g.

```
downloadUrl = '{{ api.templates }}/secure_erase.py'
```

See https://github.com/RackHD/on-tasks/blob/master/lib/task.js#L120